internal/api/schemas/loader.go

```diff
@@ -213,15 +225,15 @@ func (r *Reloader) String() string {
 }
 
 func (r *Reloader) Execute(ctx context.Context, _ context.CancelFunc) error {
+
 	select {
 	case <-ctx.Done():
 		logrus.Infof("canceled %s", r.String())
 		break
 	default:
-		for _, l := range r.Loaders {
-			if err := l.Load(); err != nil {
-				logrus.WithError(err).Errorf("unable to load a schema")
-			}
+		err := RunLoaders(r.Loaders, "model", "full")
```

Comment:
It works to set directly the string instead of the enum? cool!

Reply:
yes but we want to pass the enum type no? I just forgot actually 😅